web-extension/sources.js

```diff
@@ -3,6 +3,7 @@
     MangaHereUsPlugin:readable
     MangaNeloComPlugin:readable
     MangaKakalotComPlugin:readable
+    IsekaiScansComPlugin:readable
```

Comment:
Invalid plural, as your actual class is names `IsekaiScanComPlugin` (`Scan` instead of `Scans`)